CHANGELOG.rst

```diff
@@ -2,6 +2,26 @@
 Changelog
 =========
 
+0.11.0 (YYYY-MM-DD)
+===================
+
+* Now requires ``clear-html >= 0.4.0``.
+
+* Added :func:`zyte_common_items.processors.description_processor` and enabled
+  it for the ``description`` fields (except ``Article.description``).
+
+* Added :func:`zyte_common_items.processors.description_html_processor` and
+  enabled it for the ``descriptionHtml`` fields.
+
+* Added default implementations for the ``description`` (except
+  ``Article.description``) and ``descriptionHtml`` fields (if one of these is
```

Comment:
should it be ArticlePage and BaseArticlePage, not Article?

Reply:
Yeah, I'm using `Article` as a shorthand here and in the docs, because I don't want to list both classes :) I can change this.

Reply:
Maybe “article page classes” if you don’t want to mention both, but no strong opinion on this.

Reply:
I'm fine with "article page classes". At the same time, it also looks good to be very explicit in the changelog. 

For example,  this entry

> Added :func:`zyte_common_items.processors.description_processor` and enabled
  it for the ``description`` fields (except ``Article.description``).

can become outdated / ambiguous if we add another data type which has description field. To understand the change, one must know the current set of page objects / items at the time of the release.

That's not a big deal, I'm fine with not addressing it :) Just explaining the nit-picking over the names .